projects/Mallard/ios/Podfile.lock

```diff
@@ -505,4 +505,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6f3fb3cc2db96d753c4cb301f1618796162d6d7c
 
-COCOAPODS: 1.7.5
+COCOAPODS: 1.9.0
```

Comment:
was this a deliberate change as part of the xcode11 migration or is this just your local version of cocoapods? (just because this used to happen to me every time I ran `run-ios`)